pkg/apis/eventing/v1/test_helper.go

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	eventingduckv1 "knative.dev/eventing/pkg/apis/duck/v1"
+	messagingv1beta1 "knative.dev/eventing/pkg/apis/messaging/v1beta1"
```

Comment:
/hold
FYI, messaging v1 is on its way by @nlopezgi.
Feel free to unhold if you think we can merge with messaging v1beta1

Reply:
Yeah, this is just for tests, and really the type doesn't change, so I think we're ok. Let's see :)

Reply:
I agree with @vaikas on this, @aliok  

if you do too, please unhold